tests/validate-sigma-schema/validate.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+import sys
+from typing import Any, Dict, Iterator, List, NoReturn
+import warnings
+
+import requests
```

Comment:
```suggestion
import os
import sys
import requests
import warnings

from pathlib import Path
from typing import Any, Dict, Iterator, List, NoReturn
```

Reply:
Addressed in https://github.com/SigmaHQ/sigma/pull/4724/commits/76e24cb78ef34de8f083e873f5fd88471010cfe4.